main.go

```diff
@@ -132,10 +132,10 @@ func caduceus(arguments []string) int {
 		DeliveryInterval:    caduceusConfig.Sender.DeliveryInterval,
 		MetricsRegistry:     metricsRegistry,
 		Logger:              logger,
-		Sender: (&http.Client{
+		Sender: DoerFunc((&http.Client{
```

Comment:
```suggestion
		Sender: DoerFunc(&http.Client{
```
Do we need this additional parenthesis?

Reply:
I think the parenthesis is needed. The call breaks without it. The inner set of parentheses ensures that Do is called on the http.Client